tests/integration/test_integration_metobs.py

```diff
@@ -134,6 +113,8 @@ def test_integration_metobs_first(
         assert client.data.title == data_title
         if table:
             assert data.iloc[table_locr, table_locc] == table
+            data_header.pop("Tidsperiod (t.o.m)")
+            header_0.pop("Tidsperiod (t.o.m)")
```

Comment:
Is this due to different datetimes in the stored dataset?

Reply:
The headers contain different information to the stored one used for reference. 

Your initial solution was to take todays date and move back a set amount, but that failed when the set amount for whatever reason no longer matched the data. I tried various ways of reading the current date directly, but in the end settled for simply removing the information about the "to"-date from the comparison. The integration test hence checks that everything in the header __but__ the latest date matches, which I think is good enough.